packages/models-library/src/models_library/projects.py

```diff
@@ -202,6 +223,7 @@ class Project(BaseModel):
         example=["https://placeimg.com/171/96/tech/grayscale/?0.jpg"],
     )
     workbench: Dict[NodeID, Node]
+    ui: Optional[StudyUI]

```

Comment:
very nice!

Reply:
by setting it to Optional, it will be interpreted as ``required=False``, Nonetheless, you did not set a default value of it ... ! If you have nothing specific, just add ``None``, i.e.

```python
ui: Optional[StudyUI] = None
```

Otherwise,  it makes difficult initializing ``Project`` objects in python